scripts/qa_crowbarsetup.sh

```diff
@@ -2068,6 +2068,72 @@ function onadmin_get_ip_from_dhcp
         END{ if (res=="") exit 1; print res }' $leasefile
 }
 
+function onadmin_lonely_node_sshkey
+{
+    local lonely_ip=$1
+    wait_for 150 10 "ping -q -c 1 -w 1 $lonely_ip >/dev/null" "ping to return from ${cloud}-lonelynode" "complain 82 'could not ping lonely-node VM ($lonely_ip)'"
+
+    # wait a bit for sshd.service on ${cloud}-lonelynode
+    wait_for 10 10 "ssh_password $lonely_ip 'echo'" "ssh to be running on ${cloud}-lonelynode" "complain 82 'sshd is not responding on ($lonely_ip)'"
+
+    local pubkey=`cat /root/.ssh/id_rsa.pub`
+    ssh_password $lonely_ip "mkdir -p /root/.ssh; echo '$pubkey' >> /root/.ssh/authorized_keys"
+}
+
+function add_dns_record
+{
+    local name=$1
+    local ip=$2
+    local pfile=`get_proposal_filename dns default`
+    crowbar dns proposal show default |
+        rubyjsonparse "
+            j['attributes']['dns']['records']['$name']={};
+            j['attributes']['dns']['records']['$name']['type']='A';
+            j['attributes']['dns']['records']['$name']['values']=['$ip'];
+            puts JSON.pretty_generate(j)" > $pfile
+    crowbar dns proposal --file=$pfile edit default
+    rm -f $pfile
```

Comment:
we usually keep the pfiles to have something to look at in case of an error
so please drop the `rm`